partition/roles/monitoring/blackbox-exporter/README.md

```diff
@@ -0,0 +1,12 @@
+# blackbox-exporter
+
+Deploys the blackbox-exporter in a systemd-managed Docker container.
+
+## Variables
+
```

Comment:
What confuses me is that the role depends on `metal_partition_metal_api_addr` in `templates/config.yml` but it does not symlink to the partition defaults, which defines this variable. So, I assume if someone would just solely use this role, it would fail with an undefined variable error.

In any case, I think I would add:

```suggestion
This role uses variables from [partition-defaults](/partition). So, make sure you define them adequately as well.

```